app/(dashboard)/workouts/[id]/page.tsx

```diff
@@ -0,0 +1,100 @@
+import { User } from '../../users/interfaces/User';
+import { Workout } from '../interfaces/Workout';
+import List from './components/List';
+import api from '@/api/serverSideAxiosConfig';
+import DetailCard from '@/components/DetailCard';
+import DetailHeader from '@/components/DetailHeader';
+import { categoryToString } from '@/utils/interfaces/Category';
+import { unitToString } from '@/utils/interfaces/Unit';
+
+async function getWorkout(id: string): Promise<Workout> {
+  const { data: workout } = await api.get<Workout>(`/workouts/${id}`);
+
+  return workout;
+}
+
+async function getUsers(ids: number[]): Promise<User[]> {
+  // const { data: users } = await api.get<any[]>(`/users?ids=${ids.join(',')}`);
+  const users: User[] = [];
+  for (const id of ids) {
+    try {
+      const { data: user } = await api.get<User>(`/users/${id}`);
+      users.push(user);
+    } catch (_) {
+      console.error('User not found with id:', id);
+    }
+  }
+
+  return users;
+}
```

Comment:
This should be just one call to the backend where you pass the IDs array, and it returns an array of users. Not multiple calls to the unique ID user endpoint.

Reply:
Si, estaba esperando que aprueben https://github.com/Fiu-Fit/users-service/pull/31 para poder implementarlo

Reply:
Listo